engine/access/rpc/backend/backend_test.go

```diff
@@ -1051,58 +1071,92 @@ func (suite *Suite) TestGetNetworkParameters() {
 // TestExecutionNodesForBlockID tests the common method backend.executionNodesForBlockID used for serving all API calls
 // that need to talk to an execution node.
 func (suite *Suite) TestExecutionNodesForBlockID() {
-	if validENMap == nil {
-		validENMap = map[flow.Identifier]bool{}
-	}
-	totalBlocks := 3
```

Comment:
changed this test a little since it never really used all the blocks. So simplified it to just run with 1 block but with new test cases